10up/ElasticPress
# Facet by post type

### Description of the Change



This PR adds a facet by post type block widget similar to our Facet by Meta/Taxonomy/Meta Range ones.
Closes #3461

### How to test the Change


1.- Have a couple of post types
2.- Add a facet by post type block
3.- Go to the front end.
4.- Faceting by post type should be applied

### Changelog Entry

> Added - Facet by Post type


### Credits

Props @oscarssanchez 


### Checklist:
